flatflow/data/CMakeLists.txt

```diff
@@ -27,7 +27,10 @@ if(BUILD_TESTING)
     PRIVATE absl::str_format
     PRIVATE flatbuffers
     PRIVATE GTest::gtest_main
-    PRIVATE OpenMP::OpenMP_CXX)
+    PRIVATE OpenMP::OpenMP_CXX
```

Comment:
Unit test for ``Dataset`` does not require gRPC nor protobuf.
This change looks redundant.